common/scala/src/main/scala/org/apache/openwhisk/core/database/CouchDbRestStore.scala

```diff
@@ -74,8 +75,10 @@ class CouchDbRestStore[DocumentAbstraction <: DocumentSerializer](dbProtocol: St
   // and more documents need to be stored, then all arriving documents will be put into batches (if enabled) to avoid a long queue.
   private val maxOpenDbRequests = system.settings.config.getInt("akka.http.host-connection-pool.max-connections") / 2
 
+  private val maxRetry = loadConfigOrThrow[Int]("whisk.activation-store.retry-config.max-tries")
+  private val retryDelay = loadConfigOrThrow[FiniteDuration]("whisk.activation-store.retry-config.retry-delay")
```

Comment:
Make maxRetry and retryDelay configurable.

Reply:
looks like the section is required?